src/devices/UFireIse/UFireIse.cs

```diff
@@ -0,0 +1,415 @@
+        /// </summary>
+        /// <returns>The known value (reference value) for calibrate the high value</returns>
+        public ElectricPotential GetCalibrateHighReference()
+        {
+            return new ElectricPotential(ReadFloat(Register.ISE_CALIBRATE_REFHIGH_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling SetDualPointCalibration saves both the reading and reference value.
+        /// When there are high and low calibration points, the device will automatically use them to adjust readings.To disable dual-point adjustment, call ResetCalibration to remove all calibration data.
+        /// </summary>
+        /// <returns>The known value (reference value) for calibrate the low value</returns>
+        public ElectricPotential GetCalibrateLowReference()
+        {
+            return new ElectricPotential(ReadFloat(Register.ISE_CALIBRATE_REFLOW_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling SetDualPointCalibration saves both the reading and reference value.
+        /// When there are high and low calibration points, the device will automatically use them to adjust readings.To disable dual-point adjustment, call ResetCalibration to remove all calibration data.
+        /// </summary>
+        /// <returns>The measured value (reading value) for calibrate the high value</returns>
+        public ElectricPotential GetCalibrateHighReading()
+        {
+            return new ElectricPotential(ReadFloat(Register.ISE_CALIBRATE_READHIGH_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+
+        }
+
+        /// <summary>
+        /// Dual point uses two measures for low and high points. It needs the measured value (reading value) and the known value (reference value). Calling SetDualPointCalibration saves both the reading and reference value.
+        /// When there are high and low calibration points, the device will automatically use them to adjust readings.To disable dual-point adjustment, call ResetCalibration to remove all calibration data.
+        /// </summary>
+        /// <returns>The measured value (reading value) for calibrate the low value</returns>
+        public ElectricPotential GetCalibrateLowReading()
+        {
+            return new ElectricPotential(ReadFloat(Register.ISE_CALIBRATE_READLOW_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        ///  Resets all the stored calibration information
```

Comment:
will it use factory default after or will it be unusable until calibrated? (you mind want to clarify)